src/validation/validators/matchField/MatchFieldValidation.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { MatchFieldError } from "@/validation/errors/MatchFieldError";
+import { MatchFieldValidation } from "./MatchFieldValidation";
+import faker from "faker";
+
+const ONLY_DIGITS_REGEX = /^[0-9]*$/;
+const CAPITAL_LETTERS_REGEX = /[A-Z]+/;
+
+const makeDigitsSut = (fieldName: string): MatchFieldValidation =>
```

Comment:
I think you should create only one Sut, so, you pass the fieldname and the other params, and then call the MatchFieldValidation, once we have a generic validator